pkgs/tools/text/agrep/default.nix

```
@@ -1,33 +1,42 @@
-{ lib, stdenv, fetchFromGitHub }:
-
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+let
+  # This repository has numbered versions, but not Git tags.
+  rev = "b7d180fe73636740f694ec60c1ffab52b06e7150";
+in
 stdenv.mkDerivation {
   pname = "agrep";
   version = "3.41.5";
 
   src = fetchFromGitHub {
+    inherit rev;
     owner = "Wikinaut";
     repo = "agrep";
-    # This repository has numbered versions, but not Git tags.
-    rev = "eef20411d605d9d17ead07a0ade75046f2728e21";
-    sha256 = "14addnwspdf2mxpqyrw8b84bb2257y43g5ccy4ipgrr91fmxq2sk";
+    hash = "sha256-2J4bw5BVZgTEcIn9IuD5Q8/L+8tldDbToDefuxDf85g=";
   };
 
   # Related: https://github.com/Wikinaut/agrep/pull/11
   prePatch = lib.optionalString (stdenv.hostPlatform.isMusl || stdenv.isDarwin) ''
     sed -i '1i#include <sys/stat.h>' checkfil.c newmgrep.c recursiv.c
   '';
+
+  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" "CC+=-std=c89" ];
+
   installPhase = ''
     install -Dm 555 agrep -t "$out/bin"
     install -Dm 444 docs/* -t "$out/doc"
   '';
 
-  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
-
-  meta = with lib; {
+  meta = {
     description = "Approximate grep for fast fuzzy string searching";
     mainProgram = "agrep";
     homepage = "https://www.tgries.de/agrep/";
-    license = licenses.isc;
-    platforms = with platforms; linux ++ darwin;
+    maintainers = with lib.maintainers; [ momeemt ];
+    changelog = "https://github.com/Wikinaut/agrep/blob/${rev}/CHANGES";
+    license = lib.licenses.isc;
+    platforms = with lib.platforms; linux ++ darwin;
```

```suggestion
    platforms = lib.platforms.unix;
```

It looks like they are not restricted to Linux and Darwin.